instabot/api/api.py

```diff
@@ -614,6 +593,54 @@ def delete_comment(self, media_id, comment_id):
         url = url.format(media_id=media_id, comment_id=comment_id)
         return self.send_request(url, data)
 
+    def get_comment_likers(self, comment_id):
+        url = 'media/{comment_id}/comment_likers/?'.format(comment_id=comment_id)
+        return self.send_request(url)
+
+    def get_media_likers(self, media_id):
+        url = 'media/{media_id}/likers/?'.format(media_id=media_id)
+        return self.send_request(url)
+
+    def like_comment(self, comment_id):
+        data = self.json_data()
+        url = 'media/{comment_id}/comment_like/'.format(comment_id=comment_id)
+        return self.send_request(url, data)
+
+    def unlike_comment(self, comment_id):
+        data = self.json_data()
+        url = 'media/{comment_id}/comment_unlike/'.format(comment_id=comment_id)
+        return self.send_request(url, data)
+
+    def like(self, media_id):
+        data = self.json_data({'media_id': media_id})
+        url = 'media/{media_id}/like/'.format(media_id=media_id)
+        return self.send_request(url, data)
```

Comment:
it can be:
``` python
return self.send_request(
    url='media/{media_id}/like/'.format(media_id=media_id),
    data=self.json_data({'media_id': media_id}),
)
```

maybe will be more readable

Reply:
We can do the same thing also for the other methods (for example):
- delete_media
- comment
- . . .